server/services/communityApiService.ts

```diff
@@ -14,16 +14,38 @@ export interface DeliusUser {
 }
 
 export interface DeliusServiceUser {
-  offenderId: string
+  otherIds: OtherIds
+  offenderProfile: OffenderProfile
+  title: string
   firstName: string
   surname: string
   dateOfBirth: string
+  gender: string
+  ethnicity: string
+  religionOrBelief: string
+  disabilities: string
 }
 
 interface DeliusRole {
   name: string
 }
 
+interface OtherIds {
+  crn: string
+  nomsNumber: string
+  mostRecentPrisonerNumber: string
+}
+
+interface OffenderProfile {
+  offenderLanguages: OffenderLanguages
+}
+
+interface OffenderLanguages {
```

Comment:
similarly here, we only require primary language at this stage. we capture need for interpreter later in the form and other languages is not used anywhere else AFAIK.

Reply:
ah i see here we're just replicating the data structure used in community api. that's cool. ignore these comments.